RZBluetooth/RZBPeripheral.m

```diff
@@ -30,6 +30,12 @@ - (RZBCharacteristicBlock)notifyBlockForCharacteristicUUID:(CBUUID *)characteris
 
 - (void)setNotifyBlock:(RZBCharacteristicBlock)notifyBlock forCharacteristicUUID:(CBUUID *)characteristicUUID;
 {
+    // Fix possible crash that can occur if app attempts to clear notify blocks
+    // (ie. unregister notification/indication observers) in response to a
+    // peripheral disconnection (characteristic suddenly is nil). CSP 9/9/2016
+    if (characteristicUUID == nil) {
+        return;
+    }
```

Comment:
I think it should be a programmer error to pass nil in for characteristicUUID, since from a contract perspective it doesn't mean anything useful. The nullability annotations also enforce this. I think this should be changed to an NSParameterAssert, and the crash fixed upstream.